MCP750 PSP fails to build

**Describe the bug**

Building the current "master" branch for mcp750 yields the following build error:
```
/home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_memory.c: In function 'CFE_PSP_GetCFETextSegmentInfo':
/home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_memory.c:426: error: passing argument 1 of 'moduleFindByName' discards qualifiers from pointer target type
```

**To Reproduce**
Build "master" using standard procedure and default config.

**Expected behavior**
Build should succeed.

**System observed on:**
 - GSFC MCP750 build machine (gs582w-cfelnx)

**Additional context**
This is an error now due to the inclusion of `-Wall -Werror` in the default build.

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

Also noted a second build failure due to stricter compile flags, which should also be fixed:

```
/home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_start.c: In function 'CFE_PSP_Main':
/home/jphickey/cfecfs/github/psp/fsw/mcp750-vxworks/src/cfe_psp_start.c:132: error: cast increases required alignment of target type
```

--- a/fsw/mcp750-vxworks/src/cfe_psp_memory.c
+++ b/fsw/mcp750-vxworks/src/cfe_psp_memory.c
@@ -59,7 +59,9 @@
 /*
 ** Define the cFE Core loadable module name
 */
-#define CFE_MODULE_NAME "cfe-core.o"
+#define CFE_MODULE_NAME_DEFAULT "cfe-core.o"
+
+static char CFE_MODULE_NAME[] = CFE_MODULE_NAME_DEFAULT;
 
 
 /*

--- a/fsw/mcp750-vxworks/src/cfe_psp_start.c
+++ b/fsw/mcp750-vxworks/src/cfe_psp_start.c
@@ -98,6 +98,7 @@ void CFE_PSP_Main( void )
    uint32 reset_type;
    uint32 reset_subtype;
    char   reset_register;
+   cpuaddr memaddr;
    int32  Status;
 
 
@@ -128,8 +129,14 @@ void CFE_PSP_Main( void )
    /*
    ** Setup the pointer to the reserved area in vxWorks.
    ** This must be done before any of the reset variables are used.
+   **
+   ** Note: this uses a "cpuaddr" (integer address) as an intermediate
+   ** to avoid a warning about alignment.  The output of sysMemTop()
+   ** should be aligned to hold any data type, being the very start
+   ** of the memory space.
    */
-   CFE_PSP_ReservedMemoryPtr = (CFE_PSP_ReservedMemory_t *)sysMemTop();
+   memaddr = (cpuaddr) sysMemTop();
+   CFE_PSP_ReservedMemoryPtr = (CFE_PSP_ReservedMemory_t *) memaddr;
 
    /*
    ** Determine Reset type by reading the hardware reset register.

